openid-4-verifiable-presentations-1_0.md

```diff
@@ -2642,6 +2709,22 @@ established by [@!RFC7515].
 * Change Controller: OpenID Foundation Artifact Binding Working Group - openid-specs-ab@lists.openid.net
 * Specification Document(s): (#verifier_attestation_jwt) of this specification
 
+### client_id
+
+* Header Parameter Name: `client_id`
+* Header Parameter Description: This header contains a Client Identifier. 
+* Header Parameter Usage Location: JWS
+* Change Controller: IETF
+* Specification Document(s): [@!RFC6749]
+
+### client_metadata
+
+* Header Parameter Name: `jwt`
+* Header Parameter Description: This header contains a JWT. Processing rules MAY depend on the `typ` header value of the respective JWT. 
```

Comment:
```suggestion
* Header Parameter Name: `client_metadata`
* Header Parameter Description: This header contains Client Metadata.
```

Reply:
fixed it with the text from the parameter definition + added reference to the definition